### PR TITLE
Fix double dialog export video

### PR DESCRIPTION
**Pull request by McGiverGim**
Alternative solution to https://github.com/betaflight/blackbox-log-viewer/pull/516 that fixes https://github.com/betaflight/blackbox-log-viewer/issues/492

I don't like to add "manually" and patch the webm library, but it seems is the only solution at this moment.

In the original repo there is a discussion about how to real fix the problem: https://github.com/thenickdude/webm-writer-js/issues/31 and a new issue with more information https://github.com/thenickdude/webm-writer-js/issues/34 and a PR https://github.com/thenickdude/webm-writer-js/pull/35 that does not seem that will be merged.

If someone has some time feel free to give a better solution to this.

### CODE REVIEW

**Comment by McGiverGim** [comment]
The code duplication is in a file that I've not modified, so we can ignore it.

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**
